vllm/model_executor/layers/mamba/mamba_mixer2.py

```diff
@@ -388,9 +388,15 @@ def forward_cuda(
         # mamba2_metadata contains metadata necessary for the mamba2 triton
         # kernels to operate in continuous batching and in chunked prefill
         # modes; they are computed at top-level model forward since they
-        # are the same and reused for all mamba layers in the same iteration
+        # stay the same and reused for all mamba layers in the same iteration
         attn_metadata: AttentionMetadata = get_forward_context().attn_metadata
 
+        num_prefills = attn_metadata.num_prefills  # #requests
+        num_decodes = attn_metadata.num_decode_tokens  # #tokens==#requests
+        num_prefill_tokens = attn_metadata.num_prefill_tokens  # #tokens
```

Comment:
Could you explain what is meant by the comments at the ends of these lines?

Reply:
Comments mean the corresponding variable values are counting "number of requests" or "number of tokens"

Reply:
do you want me to change the comments?

Reply:
Yeah, it would help clarity. Thanks!

Reply:
Resolved